sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -84,26 +90,28 @@ Infer radiology insights from a patient's radiology report using a **synchronous
 - [SampleCriticalResultInferenceSync.java][ri_sync_sample]
 
 ```java com.azure.health.insights.radiologyinsights.inferradiologyinsightssync
-RadiologyInsightsInferenceResult riResults = radiologyInsightsClient.beginInferRadiologyInsights(createRadiologyInsightsRequest()).getFinalResult();
+RadiologyInsightsJob riJobResponse = radiologyInsightsClient.beginInferRadiologyInsights("job" + new Date().getTime(), createRadiologyInsightsJob()).getFinalResult();
 ```
 
 Infer radiology insights from a patient's radiology report using an **asynchronous** client.
 - [SampleCriticalResultInferenceAsync.java][ri_async_sample]
 
 ```java com.azure.health.insights.radiologyinsights.inferradiologyinsights
-PollerFlux<PollOperationDetails, RadiologyInsightsInferenceResult> asyncPoller = radiologyInsightsAsyncClient
-        .beginInferRadiologyInsights(createRadiologyInsightsRequest());
+PollerFlux<RadiologyInsightsJob, RadiologyInsightsJob> asyncPoller = radiologyInsightsAsyncClient
+        .beginInferRadiologyInsights("job" + new Date().getTime(), createRadiologyInsightsJob());
 ```
 
 Create the request.
 
 ```java com.azure.health.insights.radiologyinsights.createrequest
-private static RadiologyInsightsData createRadiologyInsightsRequest() {
+private static RadiologyInsightsJob createRadiologyInsightsJob() {
```

Comment:
Did the sdk architects approve these names? Also, did you regenerate based off of client.tsp?

Reply:
It was on their request that we changed POST to PUT I assume, so yes.